src/libpython.h

```diff
@@ -307,6 +307,10 @@ LIBPYTHON_EXTERN void (*PySys_SetArgv_v3)(int, wchar_t **);
 
 LIBPYTHON_EXTERN void (*PySys_WriteStderr)(const char *format, ...);
 
+LIBPYTHON_EXTERN PyObject* (*PyObject_CallMethod)(PyObject *o, const char *name, const char *format, ...);
+LIBPYTHON_EXTERN PyObject* (*PySequence_GetItem)(PyObject *o, Py_ssize_t i);
+LIBPYTHON_EXTERN int (*PyObject_IsTrue)(PyObject *o);
```

Comment:
As a side note, we might need to double-check whether this is available with Python 2. (Some API functions are defined as macros rather than true functions and in that case attempts to load the associated routine would fail)

Reply:
I just checked, all exist in Python 2.7
(in one case, argument types changed from `char *` to `const char *`  between two Python 3 versions, but that was all)